Jenkinsfile

```diff
@@ -152,7 +156,15 @@ def getMatrixBuilds(buildContext) {
           def cythonDesc = cythonFlag == "True" ? ", Cython": ""
           tasks["${serverVersion}, py${runtimeVersion}${cythonDesc}"] = {
             node("${OS_VERSION}") {
-              checkout scm
+              scm_variables = checkout scm
+              env.GIT_COMMIT = scm_variables.get('GIT_COMMIT')
+              env.GIT_URL = scm_variables.get('GIT_URL')
+              env.GIT_COMMIT = scm_variables.get('GIT_COMMIT')
```

Comment:
We just initialized env.GIT_COMMIT two lines above.... assuming this is just a copy/paste error?

Reply:
oops, fixed!